src/OpenTelemetry.Instrumentation.GrpcCore/ServerTracingInterceptor.cs

```diff
@@ -207,7 +207,10 @@ public ServerRpcScope(ServerCallContext context, ServerTracingInterceptorOptions
                 currentContext ?? default,
                 tags: options.AdditionalTags);
 
-            this.SetActivity(activity);
+            if (activity != null)
```

Comment:
Is it be better to change SetActivity contract to accpet null?
The check is inside already.